packages/core/src/components/Form.tsx

```diff
@@ -305,9 +309,12 @@ export default class Form<
     const liveValidate = 'liveValidate' in props ? props.liveValidate : this.props.liveValidate;
     const mustValidate = edit && !props.noValidate && liveValidate;
     const rootSchema = schema;
+    const behaviorBitFlags =
+      ('behaviorBitFlags' in props ? props.defaultFormStateBehavior : this.props.defaultFormStateBehavior) ||
```

Comment:
I think you missed this when you renamed the prop
```suggestion
      ('defaultFormStateBehavior' in props ? props.defaultFormStateBehavior : this.props.defaultFormStateBehavior) ||
```